lib/events/encryptedfile.h

```diff
@@ -5,6 +5,10 @@
 #pragma once
 
 #include "converters.h"
+#include "logging.h"
+
+#include <openssl/evp.h>
```

Comment:
Is it used?

Reply:
In the .cpp file, yes. i've moved it there